### DISCUSSION TITLE
Contriever training script & hyper-parameter values

### DISCUSSION THREAD

**Discussion by cramraj8** [castorini/pyserini#1942] [Q&A]
Is there any pre-training or fine-tuning code associated with Contriever training (also the hyper-parameters)? @thakur-nandan  @lintool

**Comment by lintool** [comment]
Hi @cramraj8 - model training is outside the scope of pyserini. For training models, we typically use tevatron.ai

**Reply by cramraj8** [reply]
Got it. I already figured that out. And got responses from Tevatron thread. Thank you!
